### ISSUE TITLE
Service center tab controls

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1907]
> Service Center tab controls do not announce to screen reader users which tab panel is currently active. Visual users can see that one color means active and the other means inactive and screen reader users should also be alerted to the difference. Recommend adding aria-selected attributes to the tab controls.

![Screen Shot 2021-10-25 at 12 41 22 PM](https://user-images.githubusercontent.com/1569463/138759683-e2bcb998-0cab-4632-bfaf-23f50313bbd1.png)

**Comment by aschmoe** [comment]
@kevindherman ran out of time on this one.  Surely possible but unclear
